build_scripts/build_linux_rpm-2-installer.sh

```diff
@@ -83,6 +84,8 @@ cd ../chia-blockchain-gui/packages/gui || exit 1
 cp package.json package.json.orig
 jq --arg VER "$CHIA_INSTALLER_VERSION" '.version=$VER' package.json > temp.json && mv temp.json package.json
 
+jq '.build.rpm.fpm |= . + ["--before-install=../../../build_scripts/assets/rpm/before-install.sh"]' package.json > temp.json && mv temp.json package.json
```

Comment:
is this needed? I would expect you either add the command line option or a change in package.json but not both. seems odd

Reply:
This is avoiding the cross repo dev work.  But yeah, it "belongs" in `package.json` in the GUI repo "presently".  But...  this all wildly needs cleaned up because of the bidirectional nature of these references.  Draft for reason added for this.

Reply:
https://github.com/Chia-Network/chia-blockchain-gui/pull/2236